src/main/java/com/sfuentes/Stationery.java

```diff
@@ -0,0 +1,50 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Stationery {
+
+  Map<String, Integer> requested;
+
+  public Stationery() {
+    this.requested = new HashMap<>();
```

Comment:
Variables usually should be nouns, here you are using a verb in past requested, I suggest rename it to request

Reply:
fixed